views/pages/AccountPage/AccountPageHeader.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+import { Typography, Button } from '@mui/material'
+import { ArrowForwardOutlined } from '@mui/icons-material'
+import { borderRadius, fontSize, lineHeight } from '../../../constants'
+import { routes } from '../../routes/routes'
+
+const AccountPageHeader = () => {
+  const navigate = useNavigate()
+
+  return (
+    <>
+      <Typography
+        variant="h3"
+        paragraph
+        sx={{
+          color: 'common.black',
+          lineHeight: lineHeight[56],
+          alignSelf: 'flex-end',
+        }}
+      >
+        Profile
+      </Typography>
+      <Button
+        size="small"
+        endIcon={<ArrowForwardOutlined />}
+        variant="outlined"
+        sx={{
+          borderRadius: borderRadius[8],
+          fontSize: fontSize[14],
+          alignSelf: 'flex-start',
+          mt: '-10px',
```

Comment:
why do we need negative margin here? Is it to vertically position the element? If so, maybe use flex positioning?

Reply:
this is to line up the letters of the button with the shoulder of the Profile text. 

<img width="956" alt="Screenshot 2023-11-17 at 11 46 03 AM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/710d0d77-a3f9-4f85-9e98-19bd8344d19a">